docs/articles/new-expensify/account-settings/Preferences.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Preferences
+description: How to manage your Expensify Preferences
+---
+# Overview
+Your Preferences in Expensify allow you to customise how you use New Expensify.
```

Comment:
```suggestion
Your Preferences in Expensify allow you to customize how you use New Expensify.
```

I think we're standardizing on American English for help docs, yeah? 😅

Reply:
Guhhhhhhh